pyiron_base/job/interactive.py

```diff
@@ -268,17 +268,17 @@ def interactive_flush(self, path="interactive", include_last_step=False):
 
     def interactive_open(self):
         """
+        Set the run mode to interactive.
 
-        Returns:
-
+        Must be called before :method:`.run()` is called.
```

Comment:
Maybe you can also say that this is the same as `job.server.run_mode.interactive = True` (I don't insist)